library/hkdf.c

```diff
@@ -18,6 +18,11 @@
  *
  *  This file is part of mbed TLS (https://tls.mbed.org)
  */
+
+/*
```

Comment:
I'm not convinced that this is needed here. Following the policy from other files, the header always contains references, while the .c file does not - and the hkdf.h header already contains a reference to RFC 5869.